src/components/markdown/markdown-parser.ts

```diff
@@ -35,7 +37,19 @@ export async function markdownToHTML(
         .use(remarkRehype, { allowDangerousHtml: true })
         .use(rehypeExternalLinks, { target: '_blank' })
         .use(rehypeRaw)
-        .use(rehypeSanitize)
+        .use(rehypeSanitize, {
+            // Allow the `style` attribute on all elements
+            attributes: {
+                '*': ['style'],
+            },
+        })
+        .use(() => {
+            return (tree: any) => {
```

Comment:
Not important but we could typehint this as a `Node` from `unist`.

Reply:
Fixed in https://github.com/Lundalogik/lime-elements/pull/2891